.devcontainer/DockerFile

```diff
@@ -0,0 +1,17 @@
+# Base image: Python with additional tools
+FROM python:3.9-slim
```

Comment:
Could you please switch this to 3.11?

Reply:
Because we want to make sure that it is working with the latest stable Python version

Reply:
@refeed Have changed them.